cypress/support/user.ts

```diff
@@ -179,9 +179,13 @@ const login = (
 };
 
 const logout = () => {
-  cy.get('[data-cy=profile-page-btn]').click();
+  localStorage.removeItem('token');
+  localStorage.removeItem('currentRole');
+  localStorage.removeItem('user');
+  localStorage.removeItem('expToken');
+  localStorage.removeItem('impersonatingUserId');
 
-  cy.get('[data-cy=logout]').click();
+  cy.visit('/');
```

Comment:
Why does this change the tests to manually clear out the session variables? I'd have thought clicking logout would mirror the way a user does it, and mean we don't need to keep the two lists of session items in sync